package.json

```diff
@@ -34,7 +34,7 @@
   "devDependencies": {
     "browserify": "11.2.0",
     "browserify-resolutions": "1.0.6",
-    "cartoassets": "CartoDB/CartoAssets#master",
+    "cartoassets": "CartoDB/CartoAssets#bd7b7db",
```

Comment:
@piensaenpixel have to revert from using master as dependency, because the [last merge](https://github.com/CartoDB/CartoAssets/commit/aa6a3d5a02cfa8f336fa650abfdb7292320f1c6b) have a [undefined variable somewhere](@piensaenpixel have to revert from using master as dependency, because the [last merge](https://github.com/CartoDB/CartoAssets/commit/aa6a3d5a02cfa8f336fa650abfdb7292320f1c6b) have a [undefined variable somewhere]):

```bash
Running "sass:dist" (sass) task
>> Undefined variable: "$cMainDark".
>>   Line 118  Column 15  node_modules/cartoassets/src/scss/cdb-components/shapes/threePoints.scss
Warning:  Use --force to continue.
```